samples/bluetooth/direct_test_mode/src/dtm.c

```diff
@@ -17,6 +18,12 @@
 #include <nrfx/helpers/nrfx_gppi.h>
 #include <nrfx_timer.h>
 
+#if defined(NRF5340_XXAA_NETWORK) || defined(NRF52833_XXAA) || defined(NRF52811_XXAA) || defined(NRF52820_XXAA)
+    #define DIRECTION_FINDING_SUPPORTED 1
+#else
+    #define DIRECTION_FINDING_SUPPORTED 0
+#endif /* defined(NRF52833_XXAA) || defined(NRF52811_XXAA) || defined(NRF52820_XXAA) */
```

Comment:
Consider moving it to the dtm_hw.h or even create a dedicated header dtm_hw_config.h

Reply:
Done. I created a new header file with those definition